src/commands/prepare.ts

```diff
@@ -119,19 +127,34 @@ This command prepares a Provider JSON metadata definition that can be used to ge
     });
 
     ux.start('Preparing provider definition');
-    const providerJson = await prepareProviderJson(
+    const providerJsonResult = await prepareProviderJson(
       {
         urlOrSource: resolved.source,
         name: resolved.name,
-        options: { quiet: flags.quiet, timeout: flags.timeout },
+        options: {
+          quiet: flags.quiet,
+          getDocs: flags.verbose,
+          timeout: flags.timeout,
+        },
       },
       { userError, ux }
     );
 
-    const providerJsonPath = await writeProviderJson(providerJson, {
-      logger,
-      userError,
-    });
+    const docs = providerJsonResult.docs
+      ? `\n{bold gray Idexed documentation:}\n${providerJsonResult.docs
```

Comment:
issue: looks like the `gray` formating option do not work:
<img width="322" alt="image" src="https://github.com/superfaceai/cli/assets/5206165/126eefc3-41f4-4abf-9f4b-5d697cb7e4c6">